ts/components/animations/AnimatedRing.tsx

```diff
@@ -0,0 +1,81 @@
+import { Millisecond } from "italia-ts-commons/lib/units";
+import * as React from "react";
+import { Animated, Easing, StyleSheet } from "react-native";
+import themeVariables from "../../theme/variables";
+
+const styles = StyleSheet.create({
+  ring: {
+    position: "absolute",
+    borderWidth: 2,
+    borderColor: themeVariables.brandPrimary
+  }
+});
+
+interface Props {
+  dimension: number;
+  duration: Millisecond;
+  interval: Millisecond;
```

Comment:
```suggestion
  startAnimationAfter: Millisecond;
```
It could be more descriptive